src/main/resources/lib/jfrog/promotionPlugins.jelly

```diff
@@ -9,7 +9,7 @@
     <tr>
         <td colspan="2"/>
         <td colspan="2">
-            <table width="100%" name="promotionPlugin" id="pluginList-container">
+            <r:blockWrapper width="100%" name="promotionPlugin" id="pluginList-container"> <!-- TODO look at the attributes here -->
```

Comment:
I think it was about the 'name' attribute which probably isn't getting passed through, I missed this due to the number of changes,

either name should be passed through or if it works without then it can be removed